third-party/realdds/include/realdds/dds-device.h

```diff
@@ -35,6 +35,7 @@ class dds_device
                                                  dds_guid const & guid,
                                                  topics::device_info const & info );
 
+    std::shared_ptr< dds_participant > const & participant() const;
```

Comment:
Why do we need to expose the participant? Logging only?

Reply:
Why not? It stores the participant, so if you hold a `device`, you can access the participant thru it.